ocdskit/oc4ids_transforms.py

```diff
@@ -0,0 +1,1010 @@
+import copy
+import datetime
+import json
+import logging
+from collections import defaultdict
+
+from jsonpointer import resolve_pointer
+from ocdsmerge.util import sorted_releases
+
+from ocdskit.combine import merge
+
+logger = logging.getLogger("ocdskit")
+
+
+def check_type(item, item_type):
+    """ Check type and if incorrect return empty version of type so that future processing works with bad data"""
+    if not isinstance(item, item_type):
+        if item:
+            logger.warn("item {} is not of type {} so skipping".format(item, item_type.__name__))
+        return item_type()
+    return item
+
+
+def cast(item, item_type):
+    """ Cast to type and if casting fails return empty version of type """
+    try:
+        return item_type(item)
+    except ValueError:
+        return item_type()
```

Comment:
Won't this result in 0-value amounts, which is not something we want? Is there a test related to this `except` branch?